site/svelte.config.js

```diff
@@ -0,0 +1,41 @@
+import adapter from '@sveltejs/adapter-node';
+
+const API_BASE = process.env.DOCS_PREVIEW ? 'http://localhost:3456' : 'https://api.svelte.dev';
```

Comment:
This `API_BASE` stuff isn't currently being used, and would just be once we rewrite the site to use the same API as the SvelteKit site, correct? Can we remove this for now?

Reply:
sure. done